doc/getting-started/apl-nuc.rst

```diff
@@ -124,7 +124,6 @@ partition. Follow these steps:
       kernel-org.clearlinux.native.4.20.2-683
       kernel-org.clearlinux.iot-lts2018-sos.4.19.13-1901141830
```

Comment:
Which Clear Linux version are we using as a reference here?

I suspect the `release` field of the kernel is of the new type (much shorter) so the kernel number looks like `kernel-org.clearlinux.iot-lts2018-sos.4.19.19-8`

Reply:
We will update one reference here, thanks

Reply:
@gvancuts done, please help check, thanks